tools/jit/gen_jit_dispatch.py

```diff
@@ -94,7 +94,7 @@ def jit_type_of(arg):
 FROM_IVALUE = {
     'Device': '{}.toDevice()',
     'Device?': '{}.toOptional<c10::Device>()',
-    'IntArrayRef': '{}.toIntList()->elements()',
+    'IntArrayRef': 'c10::impl::toArrayRef({}.toIntList()->elements())',
```

Comment:
`c10::impl` should not be showing up in this code.

Reply:
Why not? It's ok for us to use the `impl` namespace.

Reply:
fixed, `toArrayRef` is gone.